src/helpers/shared/user-login.js

```diff
@@ -36,3 +40,10 @@ export function getPermissionApi() {
 export function getAxiosInstance() {
   return axiosInstance;
 }
+
+export function getCostApi() {
+  return {
+    getResourceTypes: () => costApi.axios.get(`${COST_API_BASE}/resource-types/`),
```

Comment:
I guess the cost API does not expose these endpoints?

Reply:
Kind of, their schema is broken and and they are not using JS client.